README.md

```diff
@@ -10,6 +10,9 @@ You can install Sake from PyPI using the following command.
 python -m pip install hikari-sake -U
 ```
 
+The `hikari-yuyo[tanjun]` feature flag can be used to ensure that the installed Tanjun version
+is compatible with Sake's Tanjun adapters.
```

Comment:
Add a note that it's recommended you still have your own pinned tanjun version as this doesn't pin